makefiles/vars.inc.mk

```diff
@@ -69,7 +69,7 @@ export WERROR                # Treat all compiler warnings as errors if set to 1
 export GITCACHE              # path to git-cache executable
 export GIT_CACHE_DIR         # path to git-cache cache directory
 export FLASHER               # The command to call on "make flash".
-export FFLAGS                # The parameters to supply to FLASHER.
+# FFLAGS                	 # The parameters to supply to FLASHER.
```

Comment:
There is a tab included instead of spaces.

Reply:
fixed.